Add user-oriented file transfer logs (#463)
iterative/terraform-provider-iterative: docs/guides/getting-started.md

With spot/preemptible instances (`spot >= 0`), auto-recovery logic and persistent storage will be used to relaunch interrupted tasks.

> -> **Note:** Uploading the `workdir` may take a long time, depending on the quantity and size of the files it contains.

Reviewer: > quantity

You mean 1000 x 1kB files is significantly slower than 1 x 1MB?

Reviewer: ```suggestion
-> **Note:** A large `workdir` may take a long time to upload.
```

Author: @casperdcl, should we do something similar for `terraform destroy` or just delete the second callout?

2. Delete all the cloud resources created by `terraform apply`.

In this example, after running `terraform destroy`, the `results` directory should contain a file named `greeting.txt` with the text `Hello, World!`

> -> **Note:** Downloading the results may take a long time, depending on the quantity and size of the new/modified files inside the `output` directory.

Reviewer: ```suggestion
-> **Note:** A large `output` directory may take a long time to download.
```
